presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java

```diff
@@ -81,6 +81,7 @@ public static synchronized void start()
                 .withClusterName("TestCluster")
                 .addContactPointsWithPorts(ImmutableList.of(
                         new InetSocketAddress(HOST, PORT)))
+                .withMaxSchemaAgreementWaitSeconds(30)
```

Comment:
The default timeout is 10 seconds, which I increased to 30 seconds. If anybody has an opinion on what a reasonable timeout value should be, please comment.